### ISSUE TITLE
Passing --recursive flag with --diff always results in non-zero exit code

### ISSUE THREAD

**Issue by mtlynch** [morishitter/tidify#13]
Issue #11 actually didn't fix in the way I was expecting:

```
$ scssfmt --version
1.0.4
$ echo "foo { width: 50px; }" > /tmp/foo.scss             
$ scssfmt --recursive "/tmp/*.scss" --diff
/tmp/foo.scss
There is no difference with the original file.
$ echo $?
1
```

**Expected**: scssfmt returns zero when there are no diffs, non-zero when there are diffs
**Actual**: scssfmt *always* returns non-zero when user passes `--recursive` and `--diff` flags

**Comment by msaktype** [comment]
@mtlynch I'm sorry, I'm late. Fixed in https://github.com/morishitter/scssfmt/pull/14 . Thanks!

**Comment by mtlynch** [comment]
Thanks!

**Comment by msaktype** [comment]
I've just released v1.0.5. Please update :)
